File: mssql/php/update/mssql_update.php
#! /usr/bin/php
<?php
/*
	mssql_update.php

					Nov/18/2015

*/
// --------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
//
include "mssql_manipulate.php";
// --------------------------------------------------------------------
print "*** 開始 ***\n";
$id_in = $argv[1];
$population_in = $argv[2];

print $id_in . "\t" . $population_in . "\n";

ini_set ('display_errors',1);

$server = 'host_mssql\EG';
$user = 'sa';
$password = 'Tiger123';

$link = mssql_connect ($server,$user,$password);

if (!$link)
	{
	die('\n\nSomething went wrong while connecting to MSSQL');
	}
else
	{
	$selected = mssql_select_db ("city", $link)
		or die ("Couldn’t open database databasename\n");
	echo "connected to databasename\n";
   
	mssql_update_proc ($id_in,$population_in);
	}

mssql_close();

print "*** 終了 ***\n";
// --------------------------------------------------------------------
?>

File: mssql/csharp/update/mssql_update.cs
// -------------------------------------------------------------------
/*
	csharp/update/mssql_update.cs

					Nov/18/2015


*/
// -------------------------------------------------------------------
using	System;
using	System.Data;
using	System.Data.SqlClient;
using	System.Text;

// -------------------------------------------------------------------
class mssql_update
{
// -------------------------------------------------------------------
static void Main (string[] args)
{
	Console.WriteLine ("*** 開始 ***");

	string	id_in = args[0];
	int	population_in = int.Parse (args[1]);

	Console.WriteLine (id_in + "\t" + population_in);

	string server_mssql = @"server=host_mssql\EG;";

	string str_connect = server_mssql
			+ "uid=sa;"
			+ "pwd=Tiger123;"
			+ "database=city";

	SqlConnection connection = new SqlConnection (str_connect);

	DataTable dtable = table_manipulate.table_prepare_proc ();

	connection.Open ();

	mssql_manipulate.mssql_data_fetch_proc (connection,dtable);

	Console.WriteLine ();

	mssql_manipulate.mssql_table_update_proc
			(connection,ref dtable,id_in,population_in);

	connection.Close ();

	Console.WriteLine ("*** 終了 ***");
}

// -------------------------------------------------------------------
}

// -------------------------------------------------------------------
